specification/network/resource-manager/Microsoft.Network/2017-06-01/networkInterface.json

```diff
@@ -579,11 +607,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The network protocol this rule applies to. Possible values are: 'Tcp', 'Udp', and '*'.",
+          "description": "The network protocol this rule applies to. Possible values are: 'Tcp', 'Udp', and 'All'.",
           "enum": [
             "Tcp",
             "Udp",
-            "*"
+            "All"
```

Comment:
Please note that this is a breaking change - if you have dependent code in PowerShell or CLI please also update them.